### PR TITLE
messaging tweaks for app list and shared libraries

### PR DESCRIPTION
**Pull request by d-s**

### CODE REVIEW

**Comment by windupgithubbot1** [comment]
Can one of the admins verify this patch?

**Comment by windupgithubbot1** [comment]
Can one of the admins verify this patch?

**Comment by jsight** [comment]
@windupgithubbot1 test this please

File: tests/src/test/java/org/jboss/windup/tests/application/WindupArchitectureDuplicateTest.java [hunk 1]
```diff
@@ -112,7 +112,7 @@ private void validateApplicationList(GraphContext graphContext)
         Assert.assertEquals(582, util.getSharedStoryPoints(THIRD_APP_FILENAME));
         Assert.assertEquals(0, util.getUniqueStoryPoints(THIRD_APP_FILENAME));
 
-        Assert.assertEquals(590, util.getTotalStoryPoints("Modules Shared Across Multiple Applications"));
+        Assert.assertEquals(590, util.getTotalStoryPoints(ProjectService.SHARED_LIBS_APP_NAME));
     }
```
Comment (OndraZizka) [inline tests/src/test/java/org/jboss/windup/tests/application/WindupArchitectureDuplicateTest.java]: Right

**Pull request merged**